sdk/attestation/azure-security-attestation/samples/policy-certificates/get_policy_certificates.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief This sample demonstrates using the Attestation Service SDK for C++ to retrieve policy
+ * management certificates from an Isolated mode attestation service instance.
+ *
+ * Remember that when an attestation service instance is in isolated mode, the service is configured
+ * with a set of X.509 certificates. The GetPolicyManagementCertificates API returns a list of the
+ * existing certificates.
+ *
+ * @remark The following environment variables must be set before running the sample.
+ * - ATTESTATION_ISOLATED_URL:  Points to an Attestation Service Instance in Isolated mode.
+ * operations.
+ * - AZURE_TENANT_ID:     Tenant ID for the Azure account.
+ * - AZURE_CLIENT_ID:     The Client ID to authenticate the request.
+ * - AZURE_CLIENT_SECRET: The client secret.
+ *
+ */
+
+#include "get_env.hpp"
+
+#include "cryptohelpers.hpp"
+#include <azure/attestation.hpp>
+#include <azure/identity.hpp>
+
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+#include <thread>
+#include <vector>
+
+// cspell:: words mrsigner mrenclave mitm
+using namespace Azure::Security::Attestation;
+using namespace Azure::Security::Attestation::Models;
+using namespace std::chrono_literals;
+using namespace Azure::Core;
+
+int main()
+{
+  try
+  {
+    // create an administration client
+    auto credential = std::make_shared<Azure::Identity::ClientSecretCredential>(
+        std::getenv("AZURE_TENANT_ID"),
+        std::getenv("AZURE_CLIENT_ID"),
+        std::getenv("AZURE_CLIENT_SECRET"));
+    AttestationAdministrationClient adminClient(
+        std::getenv("ATTESTATION_ISOLATED_URL"), credential);
+
+    // Retrieve attestation response validation collateral before calling into the service.
+    adminClient.RetrieveResponseValidationCollateral();
+
+    // Retrieve the SGX Attestation Policy from this attestation service instance.
+    Azure::Response<AttestationToken<PolicyCertificateListResult>> policyCertificates
+        = adminClient.GetPolicyManagementCertificates();
+
+    std::cout << "There are " << policyCertificates.Value.Body.Certificates.size()
+              << " certificates configured on this instance." << std::endl;
+
+    std::cout << "Enumerating policy certificates:" << std::endl;
+    for (const auto& certChain : policyCertificates.Value.Body.Certificates)
+    {
+      auto x509Cert(::Cryptography::ImportX509Certificate((*certChain.CertificateChain)[0]));
```

Comment:
my want to add a comment about why index 0 at the end

Reply:
C++ arrays are 0 indexed. If you want the first entry in a C++ array, you access array[0].

Do we really need to tell our customers how arrays work in C++?

Reply:
I meant, the reason why we take the index 0.  I assume there might be more values in the index. 
For example, if CertificateChain is an array of 10 elements, why do we take the first one?  
Imagine a few years later, a developer looking for to solve a bug might wonder about that magic number `0` over there.